### PR TITLE
Jack Juxtaposition 

### PR DESCRIPTION
**Pull request by alfienguyen22**
Let's find a way to optimize the code for this problem.

### CODE REVIEW

File: venv/Juxtaposition.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import sys
```
Comment (nguyenla) [inline venv/Juxtaposition.py]: You can delete this file. It's duplicate of the file in Kattis/ folder.
You can do so by navigate to Kattis, and then remove:
```
cd Kattis
rm -f Juxtaposition.py
```

File: Kattis/Juxtaposition.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import sys
+
+n, m, t = input().split()
+n, m, t = [int(n), int(m), int(t)]
+
+total = n*m*t
+
+print(total)
+sys.exit()
```
Comment (nguyenla) [inline Kattis/Juxtaposition.py]: I don't think you need this?

File: venv/Juxtaposition.py (hunk 1 above)
Comment (alfienguyen22) [reply venv/Juxtaposition.py]: I deleted the venv file so now there is only Kattis left.

**Comment by nguyenla** [comment]
Try removing `sys.exit()` and resubmit to Kattis?

File: Kattis/Juxtaposition.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import sys
+
+n, m, t = input().split()
+n, m, t = [int(n), int(m), int(t)]
```
Comment (nguyenla) [inline Kattis/Juxtaposition.py]: A nicer way to write this is to combine the two lines into one:
```python
n, m, t = [int(x) for x in input().split()]
```

This is called "list comprehension". You can test it out locally to see if it works.
Reference:
https://www.w3schools.com/python/python_lists_comprehension.asp

**Review by nguyenla — APPROVED** [review]
Looks good to me

File: Kattis/Juxtaposition.py (hunk 3 above)
Comment (alfienguyen22) [reply Kattis/Juxtaposition.py]: I pushed both lines of codes into one and deleted the import sys

**Comment by nguyenla** [comment]
Nice! Congrats on the first merged PR 💯

**Pull request merged**